tests/filecheck/dialects/riscv/riscv_ops_parse.mlir

```diff
@@ -0,0 +1,233 @@
+builtin.module {
```

Comment:
Looks like the RUN is missing

Reply:
But also this feels like a pretty big test for this particular problem. In my understanding, we're trying to make sure that for each operation that can parse an immediate value, the Attribute has the correct bitwidth. Let's try to solve that separately? In the meantime, let's only test the RISCVOps that have interger immediate values?

Reply:
Merged into one file in newest commit.

Reply:
Many of the operations added to the test do not have immediate values, so I do not see the benefit in testing them. Could you please reduce it to only include operations with immediate values? Also, could you please add a comment explaining the purpose of the addition to the test?

Reply:
The test is not only for immediate value, but to test both generic and custom syntax and printed and parsed appropriately. I don't understand your question?

Reply:
Ok, let's merge as-is, there may be another benefit I'm not seeing right now.